src/controllers/hotkeys/Hotkey.hpp

```diff
@@ -73,21 +64,29 @@ class Hotkey
      */
     QString getCategory() const;
 
+protected:
```

Comment:
Is there any reason for this being here? :thinking: Or just a leftover?

Reply:
Leftover